injectors/index.js

```diff
@@ -77,7 +84,19 @@ const VALID_PLATFORMS = [ 'stable', 'ptb', 'canary', 'dev', 'development' ];
       '\n'
     );
     console.log('Try again with elevated permissions.');
+  } else if (e.code === 'ENOENT') {
+    console.log(
+      process.argv[2] === 'inject'
+        ? BasicMessages.PLUG_FAILED
+        : BasicMessages.UNPLUG_FAILED,
+      '\n'
+    );
+    console.log(
+      'Replugged wasn\'t able to inject itself due to missing files.',
+      '\n'
+    );
+    console.log('Try reinstalling Discord or Replugged.');
```

Comment:
Tell the user to make sure that version is installed or to specify another version.

Reply:
> Make sure that the specified platform is installed, or try again with a different platform.
How's this? I want to make sure that this one sounds clear enough.

Reply:
Have it say the actual version instead of "specified platform". And give the syntax for using another platform.